### PR TITLE
Submitting LEXIS DDL for review

### PR DESCRIPTION
**Pull request by JTA2020**

### CODE REVIEW

File: LEXIS/export_permit_detail.sql [hunk 1]
```diff
@@ -0,0 +1,80 @@
+-- SQLINES DEMO *** _DETAIL definition
```
Comment (abimichel) [inline LEXIS/export_permit_detail.sql]: These comments can be deleted

File: LEXIS/export_permit_detail.sql [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+-- SQLINES DEMO *** DBeaver CE
+-- SQLINES DEMO *** iffer from actual native database DDL
+
+-- Dr... SQLINES DEMO ***
+
+-- SQLINES DEMO *** PORT_PERMIT_DETAIL;
+
+-- SQLINES LICENSE FOR EVALUATION USE ONLY
+CREATE TABLE THE.EXPORT_PERMIT_DETAIL (
```
Comment (abimichel) [inline LEXIS/export_permit_detail.sql]: schema should be 'lexis_replication'

File: LEXIS/export_permit_detail.sql [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	OIC_APPLICATION_NUMBER BIGINT,
+	AGENT_NUMBER VARCHAR(8),
+	AGENT_LOCN_CODE VARCHAR(2),
+	ENTRY_USERID VARCHAR(30),
+	ENTRY_TIMESTAMP TIMESTAMP(0),
+	UPDATE_USERID VARCHAR(30),
+	UPDATE_TIMESTAMP TIMESTAMP(0),
+	EXPORT_PRODUCT_TYPE_CODE VARCHAR(1),
+	CONSTRAINT EPD_PK PRIMARY KEY (EXPORT_PERMIT_DETAIL_NUMBER),
+	CONSTRAINT SYS_C00233033 CHECK ("EXPORT_PERMIT_DETAIL_NUMBER" IS NOT NULL),
```
Comment (abimichel) [inline LEXIS/export_permit_detail.sql]: check constraints can be deleted

File: LEXIS/export_permit_detail.sql [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+	CONSTRAINT SYS_C00233044 CHECK ("ENTRY_USERID" IS NOT NULL),
+	CONSTRAINT SYS_C00233045 CHECK ("ENTRY_TIMESTAMP" IS NOT NULL)
+);
+-- SQLINES LICENSE FOR EVALUATION USE ONLY
+CREATE UNIQUE INDEX EPD_PK ON THE.EXPORT_PERMIT_DETAIL (EXPORT_PERMIT_DETAIL_NUMBER);
+
+
+-- SQLINES DEMO *** _DETAIL foreign keys
+
+ALTER TABLE THE.EXPORT_PERMIT_DETAIL ADD CONSTRAINT EPD_CL_FK FOREIGN KEY (CLIENT_NUMBER,CLIENT_LOCN_CODE) REFERENCES THE.CLIENT_LOCATION(CLIENT_NUMBER,CLIENT_LOCN_CODE);
```
Comment (abimichel) [inline LEXIS/export_permit_detail.sql]: foreign keys can be deleted

**Review by abimichel — CHANGES_REQUESTED** [review]
Looks great! Just a few quick edits. I updated this page so others know what formatting needs to be done: https://apps.nrs.gov.bc.ca/int/confluence/x/bpMlCw

**Comment by JTA2020** [comment]
Changes made, please advise!

**Review by abimichel — APPROVED** [review]

**Pull request merged**